include/nleobs.h

```diff
@@ -10,16 +10,18 @@
 typedef struct nle_observation {
     int action;
     int done;
-    char in_normal_game;     /* Bool indicating if other obs are set. */
-    short *glyphs;           /* Size ROWNO * (COLNO - 1) */
-    unsigned char *chars;    /* Size ROWNO * (COLNO - 1) */
-    unsigned char *colors;   /* Size ROWNO * (COLNO - 1) */
-    unsigned char *specials; /* Size ROWNO * (COLNO - 1) */
-    unsigned char *message;  /* Size 256 */
-    long *blstats;           /* NLE_BLSTATS_SIZE */
-    int *program_state;      /* NLE_PROGRAM_STATE_SIZE */
-    int *internal;           /* NLE_INTERNAL_SIZE */
-    short *inv_glyphs;       /* NLE_INVENTORY_SIZE */
```

Comment:
That's weird. Why does the diff include these lines?

Reply:
The reason is the changed space between the code and the comments. The reason of _that_ is that clang-format decided to change this space for whatever reason.